contrib/check-config.sh

```diff
@@ -23,6 +23,9 @@ if ! command -v zgrep &> /dev/null; then
 	}
 fi
 
+kernelMajor=$(uname -r | awk -F. '{print $1}')
+kernelMinor=$(uname -r | awk -F. '{print $2}')
```

Comment:
I swear I commented about this too, but I must not have hit `Comment` on it. :cry:

I think this would be more well-supported if we avoided `awk` and used native Bash features instead:
```bash
kernelVersion="$(uname -r)"
kernelMajor="${kernelVersion%%.*}"
kernelMinor="${kernelVersion#$kernelMajor.}"
kernelMinor="${kernelMinor%%.*}"
```